webstepper/package.json

```diff
@@ -7,6 +7,7 @@
         "test-cov": "jest --no-cache --coverage",
         "build-dev": "webpack --config webpack.dev.js",
         "build": "webpack --config webpack.prod.js",
+        "build-copy": "webpack --config webpack.prod.js && cp -r dist/* $PYTA_WEBSTEPPER_DIR",
```

Comment:
1. name the command `install` (following a convention from `make`, e.g. https://thoughtbot.com/blog/the-magic-behind-configure-make-make-install)
2. don't use an environment variable, and instead the user can pass an argument when invoking the command (i.e., `npm run install <path>`)